packages/service-library/src/servicelib/rabbitmq/_client.py

```diff
@@ -238,20 +248,19 @@ async def subscribe(
             )
             await delayed_queue.bind(delayed_exchange)
 
-            _consumer_tag = await self._get_consumer_tag(exchange_name)
+            consumer_tag = await self._create_consumer_tag(exchange_name)
             await queue.consume(
                 partial(_on_message, message_handler, unexpected_error_max_attempts),
                 exclusive=exclusive_queue,
-                consumer_tag=_consumer_tag,
+                consumer_tag=consumer_tag,
             )
-            output: str = queue.name
-            return output
+            return queue.name, consumer_tag
```

Comment:
Very nice trick. This allows for the consumer_tag to be local to the service where it is used and be unique. While the queue name does not change.